app/templates/components/gh-member-avatar.hbs

```diff
@@ -1,3 +1,8 @@
-<div class="flex items-center justify-center br-100" style={{this.backgroundStyle}} ...attributes>
-    <span class="gh-member-avatar-label {{this.initialsClass}}">{{this.initials}}</span>
-</div>
\ No newline at end of file
+<figure class="gh-member-gravatar" ...attributes>
+    <div class="gh-member-initials flex items-center justify-center br-100" style={{this.backgroundStyle}} ...attributes>
```

Comment:
It's generally advised to only use splattributes `(...attributes)` once inside a component template. It's intended for passing attributes specified in the component invocation onto an element (usually the outer element) inside a component. Eg.

`<GhMemberAvatar class="my-class" @member={{this.member}} />`

Would push `class="my-class"` onto every element in the component template that has `...attributes` specified. For class attributes that's maybe not so much of a problem, but if you add element modifiers it gets a bit more tricky, eg:

`<GhMemberAvatar class="my-class" @member={{this.member}} {{did-insert this.animate}} />`

Would call the `this.animate` method three times whereas it would only be expected to be called once (poor example for this specific component but hopefully you get the gist 😄).